Src/PerformanceCollector/Filtering/Implementation/Service contract/CollectionConfigurationInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Represents the entire collection configuration for this SDK.
```

Comment:
description is not descriptive. Can you please clarify what does "entire collection configuration" mean?